packages/compass-e2e-tests/tests/collection-documents-tab.test.ts

```diff
@@ -221,43 +221,45 @@ describe('Collection documents tab', function () {
   for (const maxTimeMSMode of ['ui', 'preference'] as const) {
     it(`supports maxTimeMS (set via ${maxTimeMSMode})`, async function () {
       let maxTimeMSBefore;
-      if (maxTimeMSMode === 'preference') {
-        maxTimeMSBefore = await browser.getFeature('maxTimeMS');
+      try {
```

Comment:
This is the same test, just moved inside try / finally to make sure we are cleaning up preferences even when this fails (otherwise it's hard to tell what's really failing)